db/migrate/20161010151307_create_physical_servers.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_servers do |t|
+      t.bigint :ems_id
+      t.string :name
+      t.string :type
+      t.string :uuid
```

Comment:
Can you make this uid_ems for consistency with the other tables?  UUID is an actual thing that looks like a GUID, whereas this column is more about letting the native provider pick an alternate unique indentifier, regardless of format.